ngraph/python/tests/test_onnx/test_ops_convpool.py

```diff
@@ -281,7 +281,8 @@ def test_pad_opset_1():
 
     # no paddings arttribute
     model = get_node_model("Pad", x)
-    with pytest.raises(RuntimeError):
+    from onnx.onnx_cpp2py_export.checker import ValidationError
```

Comment:
We should not put imports into functions if possible. Can we move this to the top of the file?

Reply:
It's test code and this import is needed only locally so I don't see a reason to make it globally available. I've moved the import though.